modules/n1ql/pages/advanced.adoc

```diff
@@ -17,7 +18,7 @@ endif::[]
 
 == Flex Indexes
 
-In Couchbase Server 6.6 Enterprise Edition and later, the Flex Index feature provides the ability for a N1QL query to use a Full Text Search index transparently with standard N1QL syntax.
+In Couchbase Server 6.6 Enterprise Edition and later, the Flex Index feature provides the ability for a {sqlpp} query to use a Full Text Search index transparently with standard {sqlpp} syntax.
```

Comment:
Good call, but please note there is a whole epic of tickets for this change as part of [DOC-10445](https://issues.couchbase.com/browse/DOC-10445). It was agreed in the planning for the N1QL→SLQ++ rebrand that we would make the switchover in the latest version, i.e. release/7.2, whereas this PR is for release/7.1.

Reply:
The attribute displays as N1QL in the Antora playbook for this version so this is a change that would have 0 impact, but save time in the future.